components/org.wso2.carbon.identity.oidc.dcr/src/main/java/org/wso2/carbon/identity/oidc/dcr/processor/OIDCDCRProcessor.java

```diff
@@ -54,9 +54,14 @@
         boolean isIdentityConnectDCREnabled = Boolean.parseBoolean(IdentityUtil.getProperty(ENABLE_OIDC_DCR_CONFIG));
         if (!isIdentityConnectDCREnabled) {
             if (log.isDebugEnabled()) {
-                log.debug("Identity Connect DCR Endpoint is disabled.");
+                log.debug("Identity Connect DCR endpoint was deprecated. To enable the DCR API endpoint " +
+                        "add the following config to deployment.toml file." +
+                        "[legacy_features]" +
+                        "enable_identity_connect_dcr = true");
             }
-            throw IdentityException.error(DCRException.class, ErrorCodes.FORBIDDEN.toString());
+            String errorMessage =
+                    "/identity/connect/register API was deprecated. Refer /api/identity/oauth2/dcr endpoint";
```

Comment:
let's not hard-code the /api/identity/oauth2/dcr. Instead let's add "Refer the WSO2 official documentation for more information` <link>`"

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1407/commits/515007e26b0cd144ab864d90dfd2e76cfc8f9c2c